curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa2407b521be39a3de7be1.md

```diff
@@ -7,6 +7,16 @@ dashedName: step-14
 
 # --description--
 
+To open links in a new tab, you can use the `target` attribute on the anchor (`a`) element. 
+
+The `target` attribute specifies where to open the linked document. `target="_blank"` opens the linked document in a new tab or window.
+
+Here is the basic syntax for an `a` element with a `target` attribute:
+
+```html
+<a href="https://www.freecodecamp.org" target="_blank">freeCodeCamp</a>
+```
```

Comment:
We could remove the code example and just keep the first sentence.
Or update the hint text to better show the answer.
Or keep the first sentence and update the hint text to better show the answer.

but we can't leave this step as it is now because there are way to many people coming to the forum.
https://forum.freecodecamp.org/search?q=step%2014%20%20%23html-css%20cat%20photo%20app%20after%3A2024-03-01

this tells me that they still don't understand what attributes are.
we need to find another way to help them so they better understand

Reply:
I'd do something like `<element attribute="value">content</element>` then, so we're reinforcing the syntax with the example, instead of the specific attribute.